doc/api.rst

```diff
@@ -6,16 +6,12 @@
 API Documentation
 =================
```

Comment:
so I noticed that this page is gone from the navbar. It's of course subjective but the API documentation is something I look at *pretty* often for any project. Having to do two clicks to get there is not the most optimal I would say. My suggestion would be to add it back between "Use" and "Contribute"

Reply:
+1

Reply:
I've added it back in a786be5

Reply:
@jasmainak @alexrockhill Let me know if the current solution is good for you:

<img width="708" alt="Screenshot 2020-10-21 at 09 46 13" src="https://user-images.githubusercontent.com/2046265/96689113-450ba780-1382-11eb-8aee-7d4bebc8ce06.png">


https://3262-89170358-gh.circle-artifacts.com/0/dev/index.html